From ab0899416b23c030d18963a61a2b51864bd7cef0 Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Sun, 24 Dec 2017 16:16:31 +0800
Subject: [PATCH 19/35] arm64: allwinner: h6: add PCF8563 RTC on Pine H64 board

Pine H64 board has a PCF8563 dedicated RTC connected to its R_I2C bus.

Add it to the device tree.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 arch/arm64/boot/dts/allwinner/sun50i-h6-pine-h64.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-h6-pine-h64.dts b/arch/arm64/boot/dts/allwinner/sun50i-h6-pine-h64.dts
index 9177224..e2195b0 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-h6-pine-h64.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-h6-pine-h64.dts
@@ -168,6 +168,11 @@
 		};
 	};
 
+	pcf8563: rtc@51 {
+		compatible = "nxp,pcf8563";
+		reg = <0x51>;
+		#clock-cells = <0>;
+	};
 };
 
 &uart0 {
-- 
2.7.4

